py-polars/src/conversion/mod.rs

```diff
@@ -384,11 +384,12 @@ impl FromPyObject<'_> for Wrap<DataType> {
                     "Binary" => DataType::Binary,
                     "Boolean" => DataType::Boolean,
                     "Categorical" => DataType::Categorical(None, Default::default()),
-                    "Enum" => {
-                        return Err(PyTypeError::new_err(
-                            "Enum types must be instantiated with a list of categories",
-                        ))
-                    },
+                    "Enum" => DataType::Categorical(
+                        Some(Arc::new(RevMapping::build_enum(Utf8ViewArray::new_empty(
```

Comment:
This should be `None`, not empty.

Reply:
True, I would like to rework the type a bit to have a state of 'Non-Initalized Enum'. But preferably outside this PR, so this is Ok for now

Reply:
Enum does not accept `None` yet, I think this would be better in a separate PR